front/lib/plans/plan_codes.ts

```diff
@@ -9,10 +9,12 @@ export const TRIAL_PLAN_CODE = "TRIAL_PLAN_CODE";
 // Current pro plans:
 export const PRO_PLAN_SEAT_29_CODE = "PRO_PLAN_SEAT_29";
 export const PRO_PLAN_LARGE_FILES_CODE = "PRO_PLAN_LARGE_FILES";
+export const PRO_PLAN_PLUS_SEAT_29_CODE = "PRO_PLAN_PLUS_SEAT_29";
 
 export const PRO_PLANS_CODES = [
   PRO_PLAN_SEAT_29_CODE,
   PRO_PLAN_LARGE_FILES_CODE,
+  PRO_PLAN_PLUS_SEAT_29_CODE,
```

Comment:
As discussed IRL I'm going to try to get rid of the need for this array so we don't need to maintain it when we create a plan! Creating a runner card!